src/layouts/sidebar-sidecar/utils/_local_platform-docs-mdx/index.tsx

```diff
@@ -77,6 +78,16 @@ function _defaultComponents() {
     CodeTabs,
     pre,
     a: DocsAnchor,
+    /**
+     * In /docs, we want to hide image borders by default for now,
+     * to match existing behaviour. Note that in /tutorials, we want
+     * to show image borders by default. Later we may adjust these
+     * defaults; it would likely be ideal for /docs and /tutorials
+     * to have the same default behaviour.
+     */
+    img: ({ alt, src, title }: JSX.IntrinsicElements['img']) => (
+      <Image alt={alt} src={src} title={title} noBorder={true} />
+    ),
```

Comment:
Sweet, I dig this, very on board with making return objects minimal ☺️ 

I made `makeImageElement` similar, though a bit different from `makeHeadingElement`. It returns a component rather than acting like a component. Rationale being that in the `makeHeadingElement`, I think we're doing `HEADING_LEVELS_TO_PROPS[level]` on every render, when I believe we really only want to do it once when we call ` _defaultComponents()`.

Let me know if that meets the criteria here 🙇‍♂️